src/App.scss

```diff
@@ -0,0 +1,111 @@
+.Landing {
+  display: flex;
+  flex: 1;
+  flex-wrap: wrap;
+  height: 100vh;
+}
+
+.Login {
+  align-items: center;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 11px;
+  font-weight: 600;
+  justify-content: center;
+  min-width: 400px;
+  text-align: center;
+}
+
+.landing-img-container {
+  flex: 1;
+  min-width: 400px;
+
+  & img {
+    height: 100%;
+    object-fit: cover;
+    width: 100%;
+  }
+}
+
+.app-header {
+  font-size: 20px;
+  font-weight: bold;
+  letter-spacing: 3px;
+  line-height: 27px;
+  margin-bottom: 13px;
+  margin-top: 16px;
+}
+
+.app-subtitle {
+  font-size: 18px;
+  font-weight: 600;
+  line-height: 32px;
+  margin: 0;
+}
+
+.app-description {
+  font-size: 14px;
+  font-weight: normal;
+  line-height: 26px;
+  margin: 0 28% 49px 28%;
+}
+
+.app-logo { margin-top: 70px; }
+
+.login-form {
+  display: flex;
+  flex-direction: column;
+  letter-spacing: 1.7px;
+  width: 30%;
+}
+
+.login-form-label { padding: 5px; }
+
+.login-form-input {
+  height: 20px;
+  margin-bottom: 22px;
+  min-width: 100px;
+  padding: 5px;
+}
+
+.login-form-button {
+  background-color: black;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.25);
```

Comment:
put color black in constants like `$black : #000;` and then you can use it here as `rgba($black, .25)`
Notice that the leading 0 is not necessary.

Check for any other color mentions in the scss that you have edited, I see some more occurrences but don't want to fill you up with comments.